### PR TITLE
[Utils] Installer: Improvements for plugin and platform specific package management

### PR DESCRIPTION
**Pull request by SAtacker**
- [x] https://github.com/WasmEdge/WasmEdge/issues/2201
- [X] https://github.com/WasmEdge/WasmEdge/issues/2198
- [PARTIAL] https://github.com/WasmEdge/WasmEdge/issues/2196
- [PARTIAL] https://github.com/WasmEdge/WasmEdge/issues/2193

### CODE REVIEW

**Comment by SAtacker** [comment]
@q82419 How do you propose to handle when user provides more than one of `wasi_nn-openvino` `wasi_nn-pytorch` `wasi_nn-tensorflowlite`?
CC @hydai

**Comment by q82419** [comment]
> 

Since the backends are exclusive, I think we can catch the last assigned one and print a warning message.

**Comment by SAtacker** [comment]
> > 
> 
> Since the backends are exclusive, I think we can catch the last assigned one and print a warning message.

Thanks, I'll go with your suggestion.

**Comment by SAtacker** [comment]
The CI fails due to an unrelated issue - https://github.com/WasmEdge/WasmEdge/actions/runs/3850210679/jobs/6560082736#step:18:13887

File: utils/install.py [hunk 1]
```diff
@@ -295,32 +277,34 @@ def compare(self, version2, separator=". |-", ignorecase=True):
...
-    "Linux" + "x86_64" + IMAGE_DEPS: VersionString("0.8.2"),
-    "Linux" + "amd64" + TENSORFLOW: VersionString("0.8.1"),
-    "Linux" + "amd64" + IMAGE: VersionString("0.8.1"),
-    "Linux" + "arm64" + TENSORFLOW: VersionString("0.8.1"),
-    "Linux" + "arm64" + IMAGE: VersionString("0.8.1"),
-    "Linux" + "armv8" + TENSORFLOW: VersionString("0.8.1"),
-    "Linux" + "armv8" + IMAGE: VersionString("0.8.1"),
+    "Linux" + "x86_64" + TENSORFLOW: VersionString("0.9.0"),
+    "Linux" + "x86_64" + IMAGE: VersionString("0.9.0"),
+    "Linux" + "x86_64" + IMAGE_DEPS: VersionString("0.9.0"),
```
Comment (q82419) [inline utils/install.py:282]: Since the version of `0.9.0`, the `wasmedge-image` doesn't need the deps.
You can remove all the `IMAGE_DEPS` related code.

**Review by q82419 — CHANGES_REQUESTED** [review]

File: utils/install.py (hunk 1 above)
Comment (SAtacker) [reply utils/install.py:282]: Thanks, on it.

**Comment by SAtacker** [comment]
The aarch64 is having problem with the container (or network)
https://github.com/WasmEdge/WasmEdge/actions/runs/3856029802/jobs/6571750998
Happened more than once. CC @hydai

**Comment by hydai** [comment]
> The aarch64 is having problem with the container (or network) 

We are updating the arm64 CI runner, that's why it fails. It should be fine now.

**Review by q82419 — APPROVED** [review]
LGTM. Thanks!
cc @hydai

**Review by hydai — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**